src/bootstrap/build/util.rs

```diff
@@ -16,6 +16,26 @@ use std::process::Command;
 use bootstrap::{dylib_path, dylib_path_var};
 use filetime::FileTime;
 
+#[cfg(not(windows))]
+pub fn build_path(path: &str) -> PathBuf {
+    PathBuf::from(path)
+}
+
+#[cfg(windows)]
+pub fn build_path(path: &str) -> PathBuf {
```

Comment:
The `configure` script produces MSYS path, so we need to convert them when importing `config.mk`.

I'm not sure what the logic is to convert to Windows paths, so I'd rather avoid writing that.